care/emr/api/viewsets/medication_dispense.py

```
@@ -0,0 +1,139 @@
+
+    def get_querysets(self):
+        queryset = super().get_queryset()
+
+        if self.action == "list":
+            if "location" in self.request.GET:
+                location = get_object_or_404(
+                    FacilityLocation, external_id=self.request.GET.get("location")
+                )
+
+            if "encounter" in self.request.GET:
+                encounter = get_object_or_404(
```

Ruff (0.11.9)

Local variable `encounter` is assigned to but never used

Remove assignment to unused variable `encounter`

(F841)